src/release.rs

```diff
@@ -142,8 +137,14 @@ impl BuildConfig {
         std::fs::write(self.solana_root_path.join("farf/version.yml"), content)
             .expect("Failed to write version.yml");
 
+        let label = if let Some(tag) = commit_tag {
+            tag
+        } else {
+            commit.to_string()[..8].to_string()
+        };
```

Comment:
nit: you can do this with `unwrap_or_else`
```suggestion
        let label = commit_tag.unwrap_or_else(|| commit.to_string()[..8].to_string());
```